lib/config/size.go

```diff
@@ -87,11 +87,26 @@ func CheckFreeSpace(req Size, usage fs.Usage) error {
 	if req.Percentage() {
 		freePct := (float64(usage.Free) / float64(usage.Total)) * 100
 		if freePct < val {
-			return fmt.Errorf("%f %% < %v", freePct, req)
+			return fmt.Errorf("%.1f %% < %v", freePct, req)
 		}
 	} else if float64(usage.Free) < val {
-		return fmt.Errorf("%v < %v", usage.Free, req)
+		return fmt.Errorf("%sB < %v", formatSI(usage.Free), req)
 	}
 
 	return nil
 }
+
+func formatSI(b int64) string {
+	switch {
+	case b < 1000:
+		return fmt.Sprintf("%d ", b)
+	case b < 1000*1000:
+		return fmt.Sprintf("%.1f K", float64(b)/1000)
```

Comment:
That sounds like a good thing, but I don't think it is (I tried now). It doesn't show significant zeroes, even with `%.03g` so we get `5 G` instead of `5.00 G` and it uses exponents so we get `5e+04 T` instead of `50000 T` which is unusual...

Reply:
Ah yes, right, I think I encountered similar problems in JS-land once and implemented some custom stuff to get proper significant digits because the standard library didn't facilitate it - seems `fmt` doesn't either. So one digit it is.